presto-orc/src/main/java/com/facebook/presto/orc/stream/BooleanInputStream.java

```diff
@@ -115,9 +117,20 @@ public int countBitsSet(int items)
             items -= 8;
         }
 
-        // count remaining bits
-        for (int i = 0; i < items; i++) {
-            count += nextBit() ? 1 : 0;
+        try {
+            count = IntStream.range(0, items)
+                    .map(i -> {
+                        try {
+                            return nextBit() ? 1 : 0;
+                        }
+                        catch (IOException e) {
+                            throw new UncheckedIOException(e);
+                        }
+                    })
+                    .sum();
+        }
+        catch (UncheckedIOException e) {
+            e.printStackTrace();
```

Comment:
We cannot print a stack trace to console and drop the exception.

Reply:
Are you making a change in this file just for the sake of the making a change?

Please don't do it. You are not only making it unreadable, you are also degrading performance by this change.